### PR TITLE
Add policy sub cluster resource

### PR DESCRIPTION
**Pull request by annakhm**

### CODE REVIEW

File: nsxt/resource_nsxt_policy_sub_cluster.go [hunk 1]
```diff
@@ -0,0 +1,256 @@
+/* Copyright © 2020 VMware, Inc. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (ksamoray) [reply nsxt/resource_nsxt_policy_sub_cluster.go:1]: And change VMware to Broadcom?

**Comment by ksamoray** [comment]
Code LGTM. Can we rename the resource to reflect that it's a compute sub-cluster? As from the name it can be anything

**Comment by annakhm** [comment]
> Code LGTM. Can we rename the resource to reflect that it's a compute sub-cluster? As from the name it can be anything

Renamed to `compute_sub_cluster`

**Review by ksamoray — APPROVED** [review]
Thanks!

**Pull request merged**